src/app-layout/utils/use-resize.tsx

```diff
@@ -55,15 +55,11 @@ function useResize(
     }
   };
 
-  const position = 'side';
-  const setBottomPanelHeight = () => {};
```

Comment:
Removing resize duplication allowed to remove this unused code, because drawers never resize vertically